src/main.js

```diff
@@ -68,16 +68,27 @@ if (!window.SplunkRum) {
 
     const instanceId = generateId(64);
 
+
     const cookieName = '_splunk_rum_sid';
 
-    let rumSessionId = undefined;
+    let rumSessionId = instanceId;
     const cookieSetter = function() {
       if (!document.cookie.includes(cookieName)) {
         const sessionId = generateId(128);
-        document.cookie = cookieName + '=' + sessionId + '; path=/; max-age=' + SessionTimeoutSeconds ;
+        let cookie = cookieName + '=' + sessionId + '; path=/; max-age=' + SessionTimeoutSeconds ;
+
+        if (isIframe()) {
+          cookie += ';SameSite=None; Secure';
```

Comment:
How about we start building the decision doc in an .md file? Would be great to add explanations and reasoning behind all new decision together with the code to implement them.

Reply:
Yep, good idea, I've thought about it also.

Reply:
👍